CHANGELOG.md

```diff
@@ -15,7 +17,8 @@
 
 ### Internal
 
-* Factor out internal interface ManagedObject
+* Factor out internal interface ManagedObject.
+* Use Object Store to do table initialization.
```

Comment:
Maybe it is an enhancement but object store will throw an exception with a detailed with of everything which needs to migrated.